frame/identity/src/lib.rs

```diff
@@ -490,37 +500,40 @@ decl_error! {
 		/// Sub-account isn't owned by sender.
 		NotOwned
 	}
-}
-
-decl_module! {
-	/// Identity module declaration.
-	pub struct Module<T: Config> for enum Call where origin: T::Origin {
-		/// The amount held on deposit for a registered identity.
-		const BasicDeposit: BalanceOf<T> = T::BasicDeposit::get();
-
-		/// The amount held on deposit per additional field for a registered identity.
-		const FieldDeposit: BalanceOf<T> = T::FieldDeposit::get();
-
-		/// The amount held on deposit for a registered subaccount. This should account for the fact
-		/// that one storage item's value will increase by the size of an account ID, and there will be
-		/// another trie item whose value is the size of an account ID plus 32 bytes.
-		const SubAccountDeposit: BalanceOf<T> = T::SubAccountDeposit::get();
-
-		/// The maximum number of sub-accounts allowed per identified account.
-		const MaxSubAccounts: u32 = T::MaxSubAccounts::get();
-
-		/// Maximum number of additional fields that may be stored in an ID. Needed to bound the I/O
-		/// required to access an identity, but can be pretty high.
-		const MaxAdditionalFields: u32 = T::MaxAdditionalFields::get();
-
-		/// Maxmimum number of registrars allowed in the system. Needed to bound the complexity
-		/// of, e.g., updating judgements.
-		const MaxRegistrars: u32 = T::MaxRegistrars::get();
 
-		type Error = Error<T>;
+	#[pallet::event]
+	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	pub enum Event<T: Config> {
+		/// A name was set or reset (which will remove all judgements). \[who\]
+		IdentitySet(T::AccountId),
```

Comment:
the types in the event metadata have slightly changed but I think they are both understandable by third parties.

(instead of AccountId, it is now T::AccountId, instead of `Balance`, it is now `BalanceOf<T>`, ...)
this can be fixed with the attribute metadata

Reply:
I added the metadata attribute, I don't see a reason to change something that potentially could mess with some 3rd party app